src/statistics/distributions/bernoulli.rs

```diff
@@ -72,44 +122,155 @@ impl Distribution for Bernoulli {
         }
     }
 
-    fn inv_cdf(&self, _p: f64) -> f64 {
-        todo!()
+    /// Inverse (quantile) distribution function of the Bernoulli distribution.
+    /// # Examples
+    /// ```
+    /// # use RustQuant::statistics::distributions::*;
+    ///
+    /// let bernoulli = Bernoulli::new(0.5);
+    ///
+    /// assert_eq!(bernoulli.inv_cdf(0.5), 1.0);
+    /// ```
+    fn inv_cdf(&self, p: f64) -> f64 {
+        assert!((0.0..=1.0).contains(&p));
+
+        if p < 1.0 - self.p {
+            0.0
+        } else {
+            1.0
+        }
     }
 
+    /// Mean of the Bernoulli distribution.
+    /// # Examples
+    /// ```
+    /// # use RustQuant::statistics::distributions::*;
+    ///
+    /// let bernoulli = Bernoulli::new(0.5);
+    ///
+    /// assert_eq!(bernoulli.mean(), 0.5);
+    /// ```
     fn mean(&self) -> f64 {
         self.p
     }
 
+    /// Median of the Bernoulli distribution.
+    /// # Examples
+    /// ```
+    /// # use RustQuant::statistics::distributions::*;
+    ///
+    /// let bernoulli = Bernoulli::new(0.5);
+    ///
+    /// assert_eq!(bernoulli.median(), 1.0);
+    /// ```
     fn median(&self) -> f64 {
-        todo!()
+        if self.p < 0.5 {
+            0.0
+        } else {
+            1.0
+        }
     }
 
+    /// Mode of the Bernoulli distribution.
+    /// # Examples
+    /// ```
+    /// # use RustQuant::statistics::distributions::*;
+    ///
+    /// let bernoulli = Bernoulli::new(0.5);
+    ///
+    /// assert_eq!(bernoulli.mode(), 0.0);
+    /// ```
     fn mode(&self) -> f64 {
-        todo!()
+        if self.p <= 0.5 {
+            // if p == 0.5 both 0 and 1 are modes
+            0.0
+        } else {
+            1.0
+        }
     }
 
+    /// Variance of the Bernoulli distribution.
+    /// # Examples
+    /// ```
+    /// # use RustQuant::statistics::distributions::*;
+    ///
+    /// let bernoulli = Bernoulli::new(0.5);
+    ///
+    /// assert_eq!(bernoulli.variance(), 0.25);
+    /// ```
     fn variance(&self) -> f64 {
         self.p * (1.0 - self.p)
     }
 
+    /// Skewness of the Bernoulli distribution.
+    /// # Examples
+    /// ```
+    /// # use RustQuant::statistics::distributions::*;
+    ///
+    /// let bernoulli = Bernoulli::new(0.5);
+    ///
+    /// assert_eq!(bernoulli.skewness(), 0.0);
+    /// ```
     fn skewness(&self) -> f64 {
         let p = self.p;
         ((1.0 - p) - p) / (p * (1.0 - p)).sqrt()
     }
 
+    /// Kurtosis of the Bernoulli distribution.
+    /// # Examples
+    /// ```
+    /// # use RustQuant::statistics::distributions::*;
+    ///
+    /// let bernoulli = Bernoulli::new(0.5);
+    ///
+    /// assert_eq!(bernoulli.kurtosis(), 1.0);
+    /// ```
     fn kurtosis(&self) -> f64 {
         let p = self.p;
-        (1.0 - 6.0 * p * (1.0 - p)) / (p * (1.0 - p))
+        (1.0 - 6.0 * p * (1.0 - p)) / (p * (1.0 - p)) + 3.0
```

Comment:
This is the excess kurtosis, sorry. I should probably change the function name to `excess_kurtosis`. So the +3 is not necessary.

Reply:
Got it!.  I will leave function name change from this PR. Possibly other distributions you might want to change the name.